pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -2250,6 +2245,30 @@
         "codeowner": "@grafana/alerting-squad",
         "frontend": true
       }
+    },
+    {
+      "metadata": {
+        "name": "influxdbRunQueriesInParallel",
+        "resourceVersion": "1716564259132",
+        "creationTimestamp": "2024-05-24T15:24:19Z"
+      },
+      "spec": {
+        "description": "Enables running InfluxDB Influxql queries in parallel",
+        "stage": "privatePreview",
+        "codeowner": "@grafana/observability-metrics"
+      }
+    },
+    {
+      "metadata": {
+        "name": "disableSSEDataplane",
+        "resourceVersion": "1716564259132",
+        "creationTimestamp": "2024-05-24T15:24:19Z"
+      },
+      "spec": {
+        "description": "Disables dataplane specific processing in server side expressions.",
+        "stage": "experimental",
+        "codeowner": "@grafana/observability-metrics"
+      }
```

Comment:
You might want to do something about this part

Reply:
🤔 Don't know how this got in there but I assume it's coming from me merging main to solve a conflict in this file... I'll check this.

Reply:
@tskarhed These two are actually not newly added but they were moved from above (beginning line 2206). So I'm assuming it's alright like this.